projects/admin/src/app/acquisition/routes/receipt-lines-route.ts

```diff
@@ -76,4 +78,13 @@ export class ReceiptLinesRoute extends BaseRoute implements RouteInterface {
     };
     return field;
   }
+
+  /**
+   * Pre update record
+   * @param data - Record data
+   * @returns record data
+   */
+   private _preUpdateRecord(data: any): any {
+    return this.fieldsToRemoved(data, ['is_current_budget']);
+  }
```

Comment:
Why not just use `this.fieldsToRemoved(data, ['is_current_budget']);` into the `preUpdateRecord` (Line 50) (same remark for other routes) ?

Reply:
I could. I just thought I'd put the same name of the function I'm calling in the route. There are other places where I find the name is not OK, but we can always improve.